src/main/java/me/rigamortis/seppuku/impl/gui/hud/component/PlayerPreviewComponent.java

```diff
@@ -0,0 +1,66 @@
+package me.rigamortis.seppuku.impl.gui.hud.component;
+
+import me.rigamortis.seppuku.api.gui.hud.component.DraggableHudComponent;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.renderer.OpenGlHelper;
+import net.minecraft.client.renderer.RenderHelper;
+import net.minecraft.client.renderer.entity.RenderManager;
+import net.minecraft.entity.player.EntityPlayer;
+
+/**
+ * @author cookiedragon234 10/Dec/2019
+ */
+public class PlayerPreviewComponent extends DraggableHudComponent {
+	
+	public PlayerPreviewComponent() {
+		super("Player Preview");
+	}
+	
+	@Override
+	public void render(int mouseX, int mouseY, float partialTicks) {
+		super.render(mouseX, mouseY, partialTicks);
+		
+		final Minecraft mc = Minecraft.getMinecraft();
+		EntityPlayer ent = mc.player;
+		
+		GlStateManager.pushMatrix();
+		GlStateManager.color(1, 1, 1);
+		RenderHelper.enableStandardItemLighting();
+		GlStateManager.enableAlpha();
+		GlStateManager.shadeModel(7424);
+		GlStateManager.enableAlpha();
+		GlStateManager.enableDepth();
+		GlStateManager.rotate(0, 0, 5, 0);
```

Comment:
What's this rotation for?  What does it do to the player?

Reply:
In the screenshots you can see the player is slightly rotated upwards. Was trying to fix that but it didnt work. Will remove.